code/_onclick/ai.dm

```diff
@@ -74,7 +74,7 @@
 
 	if(aicamera.in_camera_mode)
 		aicamera.camera_mode_off()
-		aicamera.captureimage(pixel_turf, usr)
+		aicamera.captureimage(pixel_turf, usr, null, aicamera.picture_size_y - 1, aicamera.picture_size_y - 1)
```

Comment:
Is this `picture_size_y` twice on purpose?

Reply:
Realistically, cyborgs just use one variable so that its a "radius", it wouldn't hurt to have this but I'll change it to the correct one for the sake of it.